recipes/wntr/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "wntr" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/michaelbynum/WNTR/archive/0b30063fdace5d6149d22b883c03def3b3b42e4a.zip
+  sha256: f70bbedb1e0208fc74a53f4e50b5c73a6b758f8cffc29e15f840caf30f2632bf
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
+    - pip
```

Comment:
It is weird that Python and pip are in build as well.  But you can fix this in the feedstock.